cli/Makefile

```
@@ -13,7 +13,7 @@ LOADER_LDFLAGS = $(JLDFLAGS) -ffreestanding -L$(build_shlibdir) -L$(build_libdir
 ifeq ($(OS),WINNT)
 LOADER_CFLAGS += -municode -mconsole -nostdlib -fno-stack-check -fno-stack-protector -mno-stack-arg-probe
 else ifeq ($(OS),Linux)
-LOADER_CFLAGS += -DGLIBCXX_LEAST_VERSION_SYMBOL=\"$(shell echo "$(CSL_NEXT_GLIBCXX_VERSION)" | cut -d'|' -f1 | sed 's/\\//g')\"
+LOADER_CFLAGS += -DGLIBCXX_LEAST_VERSION_SYMBOL=\"$(shell echo "$(CSL_NEXT_GLIBCXX_VERSION)" | cut -d'|' -f1 | sed 's/\\//g')\" -DGLIBCXX_PATH=\"$($(private_libdir)/libstdc++.so.6)\"
```

We do not use absolute paths. Also, this is a the build, not install, path that goes here.
```suggestion
LOADER_CFLAGS += -DGLIBCXX_LEAST_VERSION_SYMBOL=\"$(shell echo "$(CSL_NEXT_GLIBCXX_VERSION)" | cut -d'|' -f1 | sed 's/\\//g')\" -DGLIBCXX_PATH=\"$($(call relpath,$(build_libdir),$(build_libdir))/libstdc++.so.6)\"
```
But that relpath is obviously meaningless (it is always `.`, so this commit is simply unnecessary and should be dropped